controllers/usersignup/usersignup_controller_test.go

```diff
@@ -2241,7 +2243,7 @@ func TestUserSignupFailedToCreateDeactivationNotification(t *testing.T) {
 				"1,external": 2,
 			})
 		assert.Equal(t, "deactivated", userSignup.Labels[toolchainv1alpha1.UserSignupStateLabelKey])
-		AssertMetricsCounterEquals(t, 0, metrics.UserSignupDeactivatedTotal) // zero because state didn't change
+		AssertMetricsCounterEquals(t, 1, metrics.UserSignupDeactivatedTotal)
```

Comment:
```suggestion
		AssertMetricsCounterEquals(t, 0, metrics.UserSignupDeactivatedTotal) // zero because state didn't change
```
This would still be 0 if we set the label to deactivated only after sending the notification as mentioned in the other comment

Reply:
Fixed